redisson/src/main/java/org/redisson/codec/FstCodec.java

```diff
@@ -191,7 +191,7 @@ public FstCodec(ClassLoader classLoader, FstCodec codec) {
     }
 
     private static FSTConfiguration copy(ClassLoader classLoader, FstCodec codec) {
-        FSTConfiguration def = FSTConfiguration.createDefaultConfiguration();
+        FSTConfiguration def = codec.getConfig().deriveConfiguration();
```

Comment:
please use `codec.config.deriveConfiguration()` instead

Reply:
Oops—not sure why I thought I needed the method 🤦‍♂ .